WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -44,6 +55,14 @@ import WordPressComAnalytics
     private var didSetupView = false
     private var listentingForBlockedSiteNotification = false
 
+
+    /// Used for fetching content.
+    private lazy var displayContext: NSManagedObjectContext = {
```

Comment:
You can ditch the closure here:

`private lazy var displayContext: NSManagedObjectContext = ContextManager.sharedInstance().newMainContextChildContext()`